recipes/enkits/all/patches/cmake-install-lib-location.patch

```diff
@@ -0,0 +1,5 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -63,1 +63,1 @@
+-    install(TARGETS enkiTS DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/enkiTS")
++    install(TARGETS enkiTS DESTINATION "${CMAKE_INSTALL_PREFIX}/lib")
```

Comment:
```suggestion
+    install(TARGETS enkiTS ARCHIVE LIBRARY RUNTIME)
```

Reply:
I think you want to install the runtime in `bin`.
So 
```suggestion
+    install(TARGETS enkiTS DESTINATION RUNTIME DESTINATION bin LIBRARY DESTINATION lib ARCHIVE DESTINATION lib)
```

Reply:
Without `DESTINATION` specified, CMake uses `CMAKE_INSTALL_BINDIR` and `CMAKE_INSTALL_LIBDIR` provided by Conan.

Reply:
Oh thanks, ignore my comment then.